### PR TITLE
Fix GitHub actions CI

### PR DESCRIPTION
**Pull request by 0xB10C**
This fixes the GitHub actions CI.

### CODE REVIEW

File: src/zmqsubscriber/zmqsubscriber_test.go [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
+// waitForZMQTransaction waits for a new transaction to come in over ZMQ.
+// It returns with `didTimeout == false` as soon as a transactions comes in.
+// Otherwise it times out after the specified `timeout` and returns `didTimeout == true`.
+func waitForZMQTransaction(t *testing.T, z *ZMQSubscriber, timeout time.Duration) (didTimeout bool) {
+	select {
+	case <-z.IncomingTx:
+		return false
+	case <-time.After(timeout):
+		t.Logf("Timed out while waiting for a transaction (timeout %s)", timeout)
+		return false
```
Comment (OttoAllmendinger) [inline src/zmqsubscriber/zmqsubscriber_test.go]: `return true` according to the docs

however I think returning `true` (and `assert.IsTrue`) if something was received is more natural

File: src/zmqsubscriber/zmqsubscriber_test.go [hunk 2]
```diff
@@ -43,22 +50,21 @@ func getTestZMQSubscriber(t *testing.T, env test.TestEnv) *ZMQSubscriber {
 }
 
 func TestZMQSubscriber(t *testing.T) {
+	const waitTimeout = 3 * time.Second
```
Comment (OttoAllmendinger) [inline src/zmqsubscriber/zmqsubscriber_test.go]: his can be a toplevel const instead

File: src/zmqsubscriber/zmqsubscriber_test.go (hunk 1 above)
Comment (OttoAllmendinger) [reply src/zmqsubscriber/zmqsubscriber_test.go]: actually in future tests we probably want to look at the deserialized value so maybe this should be `&tx`/`nil` instead (with `assert.NotNil()`)

Comment (0xB10C) [reply src/zmqsubscriber/zmqsubscriber_test.go]: Good point! Addressed.

File: src/zmqsubscriber/zmqsubscriber_test.go (hunk 2 above)
Comment (0xB10C) [reply src/zmqsubscriber/zmqsubscriber_test.go]: I'd like to have as few things as possible as global. Can move it once really needed.

**Pull request merged**